xml/admin_caasp_cluster.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "generic-entities.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0"
+  xml:id="admin-caasp-cluster">
+ <!-- ============================================================== -->
+ <title>&ceph; cluster administration</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+  This chapter introduces tasks that are performed on the whole cluster.
+ </para>
+ <sect1 xml:id="admin-caasp-cluster-shutdown">
+  <title>Shutting down and restarting the whole cluster</title>
+
+  <para>
+   To shut down the whole &ceph; cluster for planned maintenance tasks, follow
+   these steps:
+  </para>
+
+  <procedure>
+   <step>
+    <para>
+     Stop all clients that are using the cluster.
+    </para>
+   </step>
+   <step>
+    <para>
+     Verify that the cluster is in a healthy state. Use the following commands:
+    </para>
+<screen>
+&prompt.cephuser;ceph status
+&prompt.cephuser;ceph health
+</screen>
+   </step>
+   <step>
+    <para>
+     Set the following OSD flags:
+    </para>
+<screen>
+    &prompt.cephuser;ceph osd set noout
+    &prompt.cephuser;ceph osd set nobackfill
+    &prompt.cephuser;ceph osd set norecover
+    </screen>
+    <tip>
+     <para>
+      In addition, you can <emphasis>pause</emphasis> the cluster. When the
```

Comment:
Why would we pause the cluster in this procedure, @shaas ?

Reply:
This is list of commands which includes pause is the suggestion from the official ceph documentation. For Rook or even K8s there is no need for that.

Reply:
then i'll remove pausing the cluster